service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/DraftOrdersControllerMidEventTest.java

```diff
@@ -55,24 +56,24 @@
     @Autowired
     private Time time;
 
-    private Document document = document();
-
-    public DraftOrdersControllerMidEventTest() {
+    DraftOrdersControllerMidEventTest() {
         super("draft-standard-directions");
     }
 
     @BeforeEach
     void setup() {
         DocmosisDocument docmosisDocument = new DocmosisDocument(SEALED_ORDER_FILE_NAME, PDF);
+        Document document = document();
 
-        given(documentGeneratorService.generateDocmosisDocument(any(), any())).willReturn(docmosisDocument);
-    }
+        given(documentGeneratorService.generateDocmosisDocument(any(AbstractDocmosisData.class), any()))
+            .willReturn(docmosisDocument);
 
-    @Test
-    void midEventShouldGenerateDraftStandardDirectionDocument() {
         given(uploadDocumentService.uploadPDF(PDF, DRAFT_ORDER_FILE_NAME))
             .willReturn(document);
+    }
 
+    @Test
+    void midEventShouldGenerateDraftStandardDirectionDocument() {
```

Comment:
Don't think that the test needs to start with `midEventShould`, just `should` should be enough

Reply:
I totally agree, however this was done in a refactoring of draft orders controller and not in this PR. I am not going address within this.